apps/varslinger-api/src/main/java/no/nav/registre/varslingerapi/config/OpenApiConfig.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.registre.varslingerapi.config;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Contact;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import io.swagger.v3.oas.models.security.SecurityRequirement;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.Arrays;
+
+import no.nav.registre.testnorge.libs.core.config.ApplicationProperties;
+
+@Configuration
+public class OpenApiConfig implements WebMvcConfigurer {
+
+    @Bean
+    public OpenAPI openApi(ApplicationProperties applicationProperties) {
+        return new OpenAPI()
+                .components(new Components().addSecuritySchemes("bearer-jwt", new SecurityScheme()
+                        .type(SecurityScheme.Type.HTTP)
+                        .scheme("bearer")
+                        .bearerFormat("JWT")
+                        .in(SecurityScheme.In.HEADER)
+                        .name("Authorization")
+                ))
+                .addSecurityItem(
+                        new SecurityRequirement().addList("bearer-jwt", Arrays.asList("read", "write")))
+                .info(new Info()
+                        .title(applicationProperties.getName())
+                        .version(applicationProperties.getVersion())
+                        .description(applicationProperties.getDescription())
+                        .termsOfService("https://nav.no")
+                        .contact(new Contact()
+                                .url("https://nav-it.slack.com/archives/CA3P9NGA2")
+                                .email("dolly@nav.no")
+                                .name("Team Dolly")
+                        )
+                        .license(new License()
```

Comment:
Skal ikke denne være `Super Strict License`: https://opensource.org/licenses/super-strict-license
Har ikke fått beskjed om det fra noen, men det er det de andre appene har hatt.

Reply:
Er du sikker på det? Trodde vi lage kode for ofentligheten

Reply:
Jau, har som sagt ikke fått beskjed om det direkte fra noen, men Super Strict License med den lenken er det som ligger i de fleste testnorge-applikasjonene er jeg ganske sikker på. Når det er sagt funker ikke lenken lenger...

Reply:
Det var det gamle systemet :)